.evergreen/start-docker-envs.sh

```diff
@@ -1,70 +1,77 @@
 #! /usr/bin/env bash
 
```

Comment:
```suggestion
```

This should never be present for scripts intended to always be executed as a bash source script rather than a standalone script

Reply:
Makes a lot of sense, thanks for highlighting!